code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -186,12 +186,14 @@
 	//Atmospherics
 /obj/item/clothing/head/helmet/space/hardsuit/engine/atmos
 	name = "atmospherics hardsuit helmet"
-	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Has thermal shielding."
+	desc = "A special helmet designed for work in a hazardous, low- or high-pressure environment. Has thermal shielding, but the same shielding makes it unsuitable for welding work."
```

Comment:
low- or high-pressure

???????

Reply:
myeh

Reply:
`extreme-pressure` is the phrase you're looking for